Header header
string parent_frame_id
geometry_msgs/Pose pose
uint32 id

uint8 UNKNOWN = 0
uint8 SMALL = 1
uint8 LARGE = 2
uint8 type

float32 length
float32 width
